scholia/app/templates/base.html

```diff
@@ -481,6 +481,48 @@
          }
      });
 
+     
+  var headings = document.querySelectorAll("h2,h3");
+  if (headings.length >= 4) {
+    var toc = document.createElement("div");
+    toc.className = "table-of-contents"
+
+    tocTitle = document.createElement("b");
+    tocTitle.innerText = "Table of Contents"
+    toc.appendChild(tocTitle)
+
+    var tocList = document.createElement("ul");
+    var sublist = "";
+    for (let i = 0; i < headings.length; i++) {
+      const element = headings[i];
+      if (!element.id) {
```

Comment:
(just a thought; no action needed)

Would be nice to have a dump of this. All headers should have one, I guess.

Reply:
What does "have a dump of this" means?

Reply:
Yeah, that was not very clear, sorry. I meant, a list of all element with/without identifiers (aka dump of this), because apparently there is enough inconsistency left that an extra `if` statement is needed.

Reply:
Would be good to get this list of headings so they can be added, however the if statement should remain for future headings that are added where people may have forgetten